package.json

```diff
@@ -39,18 +39,19 @@
   },
   "devDependencies": {
     "@types/geojson": "7946.0.8",
-    "@types/node": "^17.0.21",
-    "@typescript-eslint/eslint-plugin": "^4.8.0",
-    "@typescript-eslint/parser": "^4.8.0",
+    "@types/node": "18.11.9",
```

Comment:
Is the intention to lock the version for this particular dependency?  Just curious.

Reply:
Ran in to problems straying too far from this version. For example at time of committing 18.18.6 falls over. Could maybe loosen to patch ~ level updates, though want to understand more how the relationship works between this and whatever depends on it first.